gio/src/application.rs

```diff
@@ -12,12 +12,18 @@ use std::mem::transmute;
 
 pub trait ApplicationExtManual {
     #[doc(alias = "g_application_run")]
-    fn run(&self, argv: &[String]) -> i32;
+    fn run(&self) -> i32;
+    #[doc(alias = "g_application_run")]
+    fn run_with_args(&self, argv: &[String]) -> i32;
```

Comment:
Ugh, this is annoying :)

```suggestion
    fn run_with_args<S: AsRef<str>>(&self, argv: &[S]) -> i32;
```

please